### DISCUSSION TITLE
Big Hubs Heuristic Test this Friday the 14th/8/2020

### DISCUSSION THREAD

**Discussion by ocularvr** [mozilla/hubs#2846] [General Questions & Discussion] [answered]
Hi guys,

We will run a test for a large group on our hubs server and have formulated these questions. Please feel free to add your questions, I will happily include them and share the results here next Monday the 17th/8/2020.

We shall use a method of testing which is known as heuristic analysis. This really helps us understand how to make the system human and easy to engage in. We will ask a question and the answers will have a "range like" format. The users will answer by selecting a number from 0 to 100. Questions like did the system load fast? 0=too slow ----- 100=instant..etc. 

The overall result will fit into this equation:

**Collaborative Experience = 1 - User Effort**

User effort is the average sum of the answers whereas Collaborative Experience is what we want to achieve - a tool that allows for a meaningful online experience where we share information.

Things we will pass/fail will include:

1. How long did it take you to go from the feeling of a technological "wow" to actually feeling like Alessandro is talking and you are with him somewhere:

0 = The whole time it felt like a video game
100 = I felt like Alessandro was with me right away and my mind focused on the surroundings and his talk

2. Could you hear Alessandro (host) clearly throughout the experience?
0 = Had no audio
100 = Like he was next to me

3. Were the buttons, labels and texts on the screen easy to understand or did you not really understand the controls?
0 = Did not understand the screen
100 = Very easy to grasp

4. Was navigation in VR easy or hard? Could you move around intuitively or was it confusing?
0= I was totally lost
100 = very easy and intuitive

5. Did it make sense to use videos in the 3d environment? 
0 = Totally unnecessary
100 = Very useful

6. How was the experience in the Virtual trade show?
0 = Boring
100 = Interesting

7. How was the experience in the Photo studio
0 = Boring
100 = Interesting

8. How was the experience in the Photo gallery?
0 = Boring
100 = Interesting

9. How was the experience in the Outdoor VR seminar?
0 = Boring
100 = Interesting

10. In your own words, describe the overall value of the experience

**Comment by emclaren** [comment]
Hi @ocularvr - thanks for sharing this. I missed this discussion here. I would love to hear how your analysis went! 

If you want you can share here, or if for some reason you don't want to share your results publicly yet you can email hubs@mozilla.com

**Reply by emclaren** [reply]
Oh hey - I just saw you share results here : https://github.com/mozilla/hubs/discussions/2902 

Thanks!

**Comment by ocularvr** [comment, marked as answer]
Dear Elgin,

I updated our post to include the document link on gdrive. I have included
my partner Dr Alessandro Devigus who is the project lead. We are making a
huge effort to get things right with hubs and we understand that usability
is key. The trade show venue is by far the most challenging room we have
made, the main difficulty being making something that is low enough on
polygons to be multi platform yet rich enough to convey a meaningful
experience.

We will have a large number of people in our project in September and would
invite you to join us, after all the documentation on user experience is
actually being there :)

If we can work in any way with your team, please let me know because we
would like to include Mozilla in our virtual trade show. The idea here that
you or someone from the Mozilla hubs is there in a stand we can make with
the logo and props, then they would just interact with our subscribers.

Would this be possible?

Thanks,

Mario/Alessandro
https://dentist.camera

**Reply by emclaren** [reply]
Thanks again for sharing this - I am going to mark this as an answer to help with filtering in our github discussions for unanswered questions (this beta version of Github Discussions doesn't allow us to mark some discussions as "not in need of an answer", this feature is coming soon though).

**Comment by ocularvr** [comment]
Hi Mario,

Your event sounds very interesting, we are excited to hear how you are
using Hubs. If you are interested, we would be happy to share more
information about the work you are doing with our community (we can post in
our Discord or on Social media).

When will your event be? We unfortunately don't have the capacity to
regularly attend events, but depending on the timing, I might be able to
drop in for an hour or so to answer any questions folks might have. Please
let me know

On Wed, Aug 26, 2020 at 1:29 AM Mario Mokarram <ocularvr.com@gmail.com>
wrote:

> Dear Elgin,
>
> I updated our post to include the document link on gdrive. I have included
> my partner Dr Alessandro Devigus who is the project lead. We are making a
> huge effort to get things right with hubs and we understand that usability
> is key. The trade show venue is by far the most challenging room we have
> made, the main difficulty being making something that is low enough on
> polygons to be multi platform yet rich enough to convey a meaningful
> experience.
>
> We will have a large number of people in our project in September and
> would invite you to join us, after all the documentation on user experience
> is actually being there :)
>
> If we can work in any way with your team, please let me know because we
> would like to include Mozilla in our virtual trade show. The idea here that
> you or someone from the Mozilla hubs is there in a stand we can make with
> the logo and props, then they would just interact with our subscribers.
>
> Would this be possible?
>
> Thanks,
>
> Mario/Alessandro
> https://dentist.camera
>
> On Tue, Aug 25, 2020 at 8:42 PM elgin mclaren <notifications@github.com>
> wrote:
>
>> Hi @ocularvr <https://github.com/ocularvr> - thanks for sharing this. I
>> missed this discussion here. I would love to hear how your analysis went!
>>
>> If you want you can share here, or if for some reason you don't want to
>> share your results publicly yet you can email hubs@mozilla.com
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/mozilla/hubs/discussions/2846#discussioncomment-56607>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AQQIYYCRXGJWPMNKCAB5PNTSCQAY5ANCNFSM4P44MJCA>
>> .
>>
> --
> You received this message because you are subscribed to the Google Groups
> "hubs" group.
> To unsubscribe from this group and stop receiving emails from it, send an
> email to hubs+unsubscribe@mozilla.com.
> To view this discussion on the web visit
> https://groups.google.com/a/mozilla.com/d/msgid/hubs/CAMYwtRmy0%3DcB%3DdWdgcojd%3DbqGNLW%3DKgtOHpmtcZ9tDtR1DiNkQ%40mail.gmail.com
> <https://groups.google.com/a/mozilla.com/d/msgid/hubs/CAMYwtRmy0%3DcB%3DdWdgcojd%3DbqGNLW%3DKgtOHpmtcZ9tDtR1DiNkQ%40mail.gmail.com?utm_medium=email&utm_source=footer>
> .
>


-- 
Elgin-Skye McLaren

Community Manager

Hubs by Mozilla <https://hubs.mozilla.com>
